kafka-elastic/src/test/java/com/baeldung/kafka/service/KafkaProducerServiceLiveTest.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.kafka.service;
+
+import com.baeldung.kafka.configs.KafkaTopicConfig;
+import com.baeldung.kafka.model.NotificationModel;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.kafka.core.KafkaTemplate;
+
+import static org.mockito.Mockito.*;
+
+class KafkaProducerServiceLiveTest {
+
+    @Mock
+    private KafkaTemplate<String, NotificationModel> kafkaTemplate;
+
+    @InjectMocks
+    private KafkaProducerService kafkaProducerService;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @Test
+    void testSendMessage() {
```

Comment:
We use a given_when_then naming convention for tests (same elsewhere)

Reply:
incoprated